pom.xml

```diff
@@ -33,9 +36,9 @@
   </distributionManagement>
 
   <scm>
-    <connection>scm:git:git@github.com/kohsuke/${project.artifactId}.git</connection>
-    <developerConnection>scm:git:ssh://git@github.com/kohsuke/${project.artifactId}.git</developerConnection>
-    <url>http://${project.artifactId}.kohsuke.org/</url>
+    <connection>scm:git:git@github.com/jenkins/lib-${project.artifactId}.git</connection>
+    <developerConnection>scm:git:ssh://git@github.com/jenkinsci/lib-${project.artifactId}.git</developerConnection>
+    <url>https://${project.artifactId}.kohsuke.org/</url>
```

Comment:
This is going to be a problem. We should either delete https://github.com/timja/access-modifier/blob/8f45e2a54be18232f3f9ad0211e6673889b9ce38/pom.xml#L31-L36 and do without Maven site generation, or deploy to this repo and hope that URL still works (which I doubt).

Reply:
this site doesn't even exist, https://access-modifier.kohsuke.org/

I repointed it to github in the incrementals PR